tests/backend/topology/test_pyramid.py

```diff
@@ -9,9 +9,9 @@
 from pyswarms.backend.topology import Pyramid
 
 
-def test_compute_gbest_return_values(swarm):
```

Comment:
We can parameterize this such that:

```python
@pytest.mark.parameterize("static", [True, False])
def test_compute_gbest_return_values(swarm, static):
      """Test if compute_gbest() gives the expected return values"""
      topology = Pyramid()
      topology = Pyramid(static=static)
      expected_cost = 1
      expected_pos = np.array([1, 2, 3])
      pos, cost = topology.compute_gbest(swarm)
      assert cost == expected_cost
      assert (pos == expected_pos).all()
```

So we can eliminate the need for the static fixture. Usually we use fixtures to create mockups: classes, dictionaries, etc. If we're just changing the parameters, we can simply parameterize this easily :100: Hail pytest!

Reply:
I thought a fixture might be cleaner because I'd have to write `@pytest.mark.parametrize("static", [True, False])` above every function 😄.